src/screens/Manager/DeviceCard/DeviceAppStorage.js

```diff
@@ -0,0 +1,143 @@
+import React, { memo } from "react";
+
+import { View, StyleSheet } from "react-native";
+import * as Animatable from "react-native-animatable";
+import { Trans } from "react-i18next";
+
+import type { AppsDistribution } from "@ledgerhq/live-common/lib/apps";
+import { formatSize } from "@ledgerhq/live-common/lib/apps";
+import LText from "../../../components/LText";
+import colors from "../../../colors";
+
+import Warning from "../../../icons/Warning";
+
+type Props = {
+  distribution: AppsDistribution,
+};
+
+const DeviceAppStorage = ({
+  distribution: {
+    freeSpaceBytes,
+    appsSpaceBytes,
+    shouldWarnMemory,
+    totalAppsBytes,
+    apps,
+  },
+}: Props) => {
+  const installedApps = apps.filter(Boolean);
+  const appSizes = installedApps.map(({ bytes, currency }) => ({
+    ratio: Number((bytes / appsSpaceBytes) * 100).toFixed(2),
+    color: (currency && currency.color) || "#000000",
+  }));
+
+  const storageWarnStyle = {
+    color: shouldWarnMemory ? colors.alert : colors.darkBlue,
+  };
+
+  return (
+    <View style={styles.root}>
+      <View style={[styles.row, styles.storageLeftRow]}>
+        <LText bold style={styles.storageText}>
+          <Trans i18nKey="manager.storage.title" />
+        </LText>
+        <View style={{ flexDirection: "row" }}>
+          {shouldWarnMemory && <Warning color={colors.alert} size={15} />}
+          <LText bold style={[styles.storageText, storageWarnStyle]}>
+            {" "}
+            {formatSize(freeSpaceBytes) || "0kb"}
```

Comment:
can you find a way to solve this in live-common? it sounds like maybe we want an option param to say we want to render the 0 case.
in future we will need to i18n it so that's good if we centralize at same place.